gui/mountPrimaryWindow.py

```diff
@@ -114,6 +115,23 @@ def mountBucket(self):
                 self.errorMessageBox(f"Error: Cloudfuse needs to create the directory {directory}, but it already exists!")
                 return
             
+            # do a dry run to validate options and credentials
+            commandParts = ['cloudfuse.exe', 'mount', directory, f'--config-file={configPath}', '--dry-run']
```

Comment:
Is a button needed to do a dry-run, or is the intent to do a dry-run every time the mount button is pressed?

Reply:
Here, the intent is to run it every time, as a validation step before mounting. This should eliminate a bunch of silent failures.
If we want to have a button that does this, that's definitely an option. I think it would be nice for the GUI to have a "check" button in the config window, for example, but that's not urgent.